src/chart/FtpHistoryChart.svelte

```diff
@@ -0,0 +1,80 @@
+<script>
+    import { onMount } from "svelte";
+    import Chart from "chart.js";
+    import Card from "../components/Card.svelte";
+    import * as ChartAnnotation from "chartjs-plugin-annotation";
+    import { ftpAverageOutputs } from "../store/store.js";
+    import moment from "moment";
+
+    const getPlotPoints = (data) => {
+        return data.map((ride) => {
+            // TODO (tmack) find why there are duplicates?
+            console.log(ride);
```

Comment:
I need to clean something up here. Thought I fixed this, but upon cleaning things up for PR I removed the `ftpAverageOutputs` method from `../store/store.js`. I'll resolve that tonight.

Reply:
fixed in the second commit https://github.com/jasonlutterloh/metrics-peloton/pull/8/commits/e9554d11d0676837df367d9e49cc3f5ee0dbe6db (accidentally removed the store method for accessing averageOutput (ie. "Avg. Watts") from the data file.